skglm/estimators.py

```diff
@@ -1675,3 +1691,201 @@ def fit(self, X, y):
             verbose=self.verbose)
 
         return _glm_fit(X, y, self, quad_group, group_penalty, solver)
+
+####################
+# WIP Graphical Lasso
+####################
+
+
+class GraphicalLasso():
```

Comment:
the file is already quite big with 1700 lines; since graphical lasso is different from other estimators maybe this class could go in its own `covariance.py` file ?